quickstart.md

```diff
@@ -47,13 +47,13 @@ To ensure that files will be excluded from commits you can use the following git
 Please run the following command to start the Mudita Center dev environment:
 
 ```bash
-npm run develop
+npm run app:develop
 ```
 
 This will start the application with hot-reload so you can instantly start developing it. You can also enable logs by executing:
 
 ```bash
-npm run develop -- --stream
+npm run app:develop -- --stream
```

Comment:
Thanks for updating the documentation. Some commands are outdated. Now, npm run app
is sufficient without -- --stream.
```suggestion
```

Please remove this entire section 🙏

Reply:
Should I delete this?
```bash
lerna run --stream --scope @mudita/mudita-center-app dev:start:renderer
lerna run --stream --scope @mudita/mudita-center-app dev:start:main
```

Reply:
yes, you can, thx